xls/dslx/stdlib/std.x

```diff
@@ -396,6 +396,41 @@ fn round_up_to_nearest_test() {
     assert_eq(u32:4, round_up_to_nearest(u32:4, u32:2));
 }
 
+// Returns `x` rounded up to `y`, where `y` is a positive power of 2
+fn round_up_to_pow2_unsigned<N: u32>(x: uN[N], y: uN[N]) -> uN[N] { (x + y - uN[N]:1) & -y }
```

Comment:
can you try to run https://github.com/google/xls/tree/main/xls/dslx/fmt on this?

Reply:
This code has already been formatted with `dslx_fmt`